users/schema.py

```diff
@@ -0,0 +1,76 @@
+import graphene
+from graphene import Node
+from graphene_django import DjangoObjectType, DjangoConnectionField
+from graphene_django_cud.mutations import (
+    DjangoPatchMutation,
+    DjangoDeleteMutation,
+    DjangoCreateMutation,
+)
+
+from users.models import (
+    User,
+)
+
+
+class UserNode(DjangoObjectType):
+    class Meta:
+        model = User
+        interfaces = (Node,)
+        exclude_fields = ("password",)
+
+    full_name = graphene.String(source="get_full_name")
+    initials = graphene.String()
+    profile_picture = graphene.String()
+
+    def resolve_profile_picture(self: User, info, **kwargs):
+        return self.profile_image_url
+
+    def resolve_initials(self: User, info, **kwargs):
```

Comment:
This particular initials implementation can break, unfortunately.

A better version (here taken from a User model):

```
  @property
  def initials(self):
      full_name = self.get_full_name() or ""

      if len(full_name) < 2:
          full_name = self.username

      if not " " in full_name:
          return full_name[0:2].upper()
      else:
          split = re.split("\W+", full_name)
          first_part, last_part = split[0], split[-1]
          return f"{first_part[0].upper()}{last_part[0].upper()}"
```